packages/@aws-cdk/aws-kinesisanalyticsv2/test/flink-application.test.ts

```diff
@@ -0,0 +1,528 @@
+import { arrayWith, objectLike, ResourcePart } from '@aws-cdk/assert';
+import '@aws-cdk/assert/jest';
+import * as iam from '@aws-cdk/aws-iam';
+import * as s3 from '@aws-cdk/aws-s3';
+import * as core from '@aws-cdk/core';
+import * as path from 'path';
+import * as ka from '../lib';
+
+function buildStack() {
+  const stack = new core.Stack();
+  const bucket = new s3.Bucket(stack, 'CodeBucket');
+  const requiredProps = {
+    runtime: ka.FlinkRuntime.FLINK_1_11,
+    code: ka.ApplicationCode.fromBucket(bucket, 'my-app.jar'),
+  };
+
+  return { stack, bucket, requiredProps };
+}
```

Comment:
It's been awhile since I've worked with this "distant context" style of test setup and it brings back bad rspec memories :) Moved this back to the style that seems to be the convention in CDK 👍